frontend/test/metabase/scenarios/visualizations/reproductions/18996-table-image-pagination.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import { restore, visitDashboard } from "__support__/e2e/cypress";
+
+const questionDetails = {
+  name: "18996",
+  native: {
+    query: `
+select 1 "ID", 'https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg' "IMAGE", 123 "PRICE"
+union all select 2, 'https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg', 123
+union all select 3, 'https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg', 123
+union all select 4, 'https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg', 123
+union all select 5, null, 123
+union all select 6, '', 123
+union all select 7, 'non-exisiting', 123
+union all select 8, null, 123
+union all select 9, 'https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg', 123
+union all select 10, 'https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg', 123
```

Comment:
`https://upload.wikimedia.org/wikipedia/commons/2/28/JPG_Test.jpg` is 20MB, I wonder if the file size may affect the CI?